### PR TITLE
feat(issues): Add organization tag values component

### PR DESCRIPTION
**Pull request by lynnagara**
`shared/groupTagValues` is mostly copy + paste from project version (minus environment filtering code)

### CODE REVIEW

File: src/sentry/static/sentry/app/views/groupDetails/shared/groupTagValues.jsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
+/*eslint react/jsx-key:*/
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/groupDetails/shared/groupTagValues.jsx]: Do you know if this is needed?

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/groupDetails/shared/groupTagValues.jsx]: It was, updated the line that needed it

File: src/sentry/static/sentry/app/views/groupDetails/organization/groupTagValues.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import GroupTagValues from '../shared/groupTagValues';
+
+export default class OrganizationGroupTagValues extends React.Component {
+  render() {
+    const query = {};
+    return <GroupTagValues {...this.props} query={query} />;
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/groupDetails/organization/groupTagValues.jsx]: How will the environment selection filter its way down to here?

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/groupDetails/organization/groupTagValues.jsx]: The entire query object is passed down via the `query` prop which allows the org/project variants to do different types of filtering.

**Review by markstory — APPROVED** [review]

**Pull request merged**